stories/core/ComboBox.stories.tsx

```diff
@@ -392,13 +398,19 @@ export const WithStatus: Story<Partial<ComboBoxProps<string>>> = (args) => {
   const options = React.useMemo(() => countriesList, []);
 
   return (
-    <ComboBox
-      options={options}
-      inputProps={{ placeholder: 'Select a country' }}
-      onChange={(value: string) => action(value ?? '')()}
-      status='negative'
-      {...args}
-    />
+    <InputGroup
+      status={args.status ?? 'negative'}
+      message='Country is required.'
+    >
+      <Label>Country</Label>
```

Comment:
Add `htmlFor` here and `id` to inputProps. See the label story for example

Reply:
yeah, good catch. added